File: leetcode-daily-2-keys-keyboard/solution/Solution.swift
// Solution in Swift

class Solution {
    /**
     * Returns the minimum number of operations required to get n 'A's on the screen
     * - parameter n: the number of characters to get on the screen
     * - returns: the minimum number of operations required
     */
    func minSteps(_ n: Int) -> Int {
        // Get the prime factors of n
        let primeFactors = getPrimeFactors(n)
        
        // Sum up the prime factors to get the minimum number of operations
        return primeFactors.reduce(0, +)
    }
    
    /**
     * Returns an array of prime factors for a given number
     * - parameter number: the number to find prime factors for
     * - returns: an array of prime factors
     */
    private func getPrimeFactors(_ number: Int) -> [Int] {
        var factors: [Int] = []
        var divisor = 2
        var num = number
        
        while num > 1 {
            // If the number is not divisible by the current divisor, increment the divisor
            if num % divisor != 0 {
                divisor += 1
            } else {
                // If the number is divisible by the current divisor, add it to the factors array
                // and divide the number by the current divisor
                factors.append(divisor)
                num /= divisor
            }
        }
        
        return factors
    }
}

File: leetcode-daily-2-keys-keyboard/solution/Solution.php
<?php // Solution in PHP

class Solution
{

    /**
     * Returns the minimum number of operations required to get n 'A's on the screen
     * @param Integer $n - the number of characters to get on the screen
     * @return Integer - the minimum number of operations required
     */
    function minSteps($n)
    {
        // Get the prime factors of $n
        $primeFactors = $this->getPrimeFactors($n);

        // Sum up the prime factors to get the minimum number of operations
        return array_sum($primeFactors);
    }

    /**
     * Returns an array of prime factors for a given number
     * @param Integer $number - the number to find prime factors for
     * @return array - an array of prime factors
     */
    private function getPrimeFactors($number)
    {
        $factors = [];

        $divisor = 2;

        while ($number > 1) {
            // If the number is not divisible by the current divisor, increment the divisor
            if ($number % $divisor !== 0) {
                $divisor++;
            } else {
                // If the number is divisible by the current divisor, add it to the factors array
                // and divide the number by the current divisor
                $factors[] = $divisor;
                $number = $number / $divisor;
            }
        }

        return $factors;
    }
}